<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="rxTab">
       <attribute name="title">
        <string>Receiver</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QGroupBox" name="servoSettingsGroupBox">
            <property name="title">
             <string>Servo Settings</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_3">
             <item>
              <widget class="QLabel" name="label">
               <property name="text">
                <string>Servo Output Rate</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="servoRateComboBox">
               <item>
                <property name="text">
                 <string>50Hz</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>100Hz</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>200Hz</string>
                </property>
               </item>
              </widget>
             </item>
             <item>
              <spacer name="verticalSpacer">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="sbusSettingsGroupBox">
            <property name="title">
             <string>S.Bus settings</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_2">
             <item>
              <widget class="QCheckBox" name="sbusEnableCheckBox">
               <property name="text">
                <string>Enable S.Bus</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_2">
               <property name="text">
                <string>Number of S.Bus outputs</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSpinBox" name="sbusOutputsSpinBox">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <property name="maximum">
                <number>4</number>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_3">
               <property name="text">
                <string>S.Bus output period</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSpinBox" name="sbusPeriodSpinBox">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <property name="suffix">
                <string>ms</string>
               </property>
               <property name="minimum">
                <number>7</number>
               </property>
               <property name="maximum">
                <number>20</number>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="verticalSpacer_2">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QGroupBox" name="channelsGroupBox">
          <property name="title">
           <string>Channels</string>
          </property>
          <layout class="QGridLayout" name="gridLayout">
           <item row="1" column="3">
            <widget class="QProgressBar" name="progressBar_8">
             <property name="maximum">
              <number>2047</number>
             </property>
             <property name="value">
              <number>0</number>
             </property>
             <property name="format">
              <string>%v</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>Ch. 1 (Thr)</string>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="label_7">
             <property name="text">
              <string>Ch. 4 (Rud)</string>
             </property>
            </widget>
           </item>
           <item row="4" column="1">
            <widget class="QProgressBar" name="progressBar_5">
             <property name="maximum">
              <number>2047</number>
             </property>
             <property name="value">
              <number>0</number>
             </property>
             <property name="format">
              <string>%v</string>
             </property>
            </widget>
           </item>
           <item row="5" column="1">
            <widget class="QProgressBar" name="progressBar_6">
             <property name="maximum">
              <number>2047</number>
             </property>
             <property name="value">
              <number>0</number>
             </property>
             <property name="format">
              <string>%v</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QProgressBar" name="progressBar_4">
             <property name="maximum">
              <number>2047</number>
             </property>
             <property name="value">
              <number>1024</number>
             </property>
             <property name="format">
              <string>%v</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QProgressBar" name="progressBar">
             <property name="maximum">
              <number>2047</number>
             </property>
             <property name="value">
              <number>1024</number>
             </property>
             <property name="invertedAppearance">
              <bool>false</bool>
             </property>
             <property name="format">
              <string>%v</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QProgressBar" name="progressBar_3">
             <property name="maximum">
              <number>2047</number>
             </property>
             <property name="value">
              <number>1024</number>
             </property>
             <property name="format">
              <string>%v</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QProgressBar" name="progressBar_2">
             <property name="maximum">
              <number>2047</number>
             </property>
             <property name="value">
              <number>1024</number>
             </property>
             <property name="format">
              <string>%v</string>
             </property>
            </widget>
           </item>
           <item row="5" column="0">
            <widget class="QLabel" name="label_9">
             <property name="text">
              <string>Ch. 6</string>
             </property>
            </widget>
           </item>
           <item row="0" column="3">
            <widget class="QProgressBar" name="progressBar_7">
             <property name="maximum">
              <number>2047</number>
             </property>
             <property name="value">
              <number>0</number>
             </property>
             <property name="format">
              <string>%v</string>
             </property>
            </widget>
           </item>
           <item row="4" column="0">
            <widget class="QLabel" name="label_8">
             <property name="text">
              <string>CH. 5</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_6">
             <property name="text">
              <string>Ch. 3 (Ele)</string>
             </property>
            </widget>
           </item>
           <item row="5" column="3">
            <widget class="QProgressBar" name="progressBar_12">
             <property name="maximum">
              <number>2047</number>
             </property>
             <property name="value">
              <number>0</number>
             </property>
             <property name="format">
              <string>%v</string>
             </property>
            </widget>
           </item>
           <item row="3" column="3">
            <widget class="QProgressBar" name="progressBar_10">
             <property name="maximum">
              <number>2047</number>
             </property>
             <property name="value">
              <number>0</number>
             </property>
             <property name="format">
              <string>%v</string>
             </property>
            </widget>
           </item>
           <item row="2" column="3">
            <widget class="QProgressBar" name="progressBar_9">
             <property name="maximum">
              <number>2047</number>
             </property>
             <property name="value">
              <number>0</number>
             </property>
             <property name="format">
              <string>%v</string>
             </property>
            </widget>
           </item>
           <item row="4" column="3">
            <widget class="QProgressBar" name="progressBar_11">
             <property name="maximum">
              <number>2047</number>
             </property>
             <property name="value">
              <number>0</number>
             </property>
             <property name="format">
              <string>%v</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>Ch. 2 (Ail)</string>
             </property>
            </widget>
           </item>
           <item row="0" column="2">
            <widget class="QLabel" name="label_10">
             <property name="text">
              <string>Ch. 7</string>
             </property>
            </widget>
           </item>
           <item row="1" column="2">
            <widget class="QLabel" name="label_11">
             <property name="text">
              <string>Ch. 8</string>
             </property>
            </widget>
           </item>
           <item row="2" column="2">
            <widget class="QLabel" name="label_12">
             <property name="text">
              <string>Ch. 9</string>
             </property>
            </widget>
           </item>
           <item row="3" column="2">
            <widget class="QLabel" name="label_13">
             <property name="text">
              <string>Ch. 10</string>
             </property>
            </widget>
           </item>
           <item row="4" column="2">
            <widget class="QLabel" name="label_14">
             <property name="text">
              <string>Ch. 11</string>
             </property>
            </widget>
           </item>
           <item row="5" column="2">
            <widget class="QLabel" name="label_15">
             <property name="text">
              <string>Ch. 12</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Orientation</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>Wing Type</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="tab_4">
       <attribute name="title">
        <string>Gyro Settings</string>
       </attribute>
      </widget>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="connectPushButton">
        <property name="text">
         <string>Connect</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="loadPushButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Load</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="savePushButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Save</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen_Configuration"/>
    <addaction name="actionSave_Configuration"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools</string>
    </property>
    <addaction name="actionSelf_Test"/>
    <addaction name="actionUpdate_Firmware"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout_RX12G_Configure"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuTools"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen_Configuration">
   <property name="text">
    <string>Open Configuration</string>
   </property>
  </action>
  <action name="actionSave_Configuration">
   <property name="text">
    <string>Save Configuration</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="actionSelf_Test">
   <property name="text">
    <string>Self Test</string>
   </property>
  </action>
  <action name="actionUpdate_Firmware">
   <property name="text">
    <string>Update Firmware</string>
   </property>
  </action>
  <action name="actionAbout_RX12G_Configure">
   <property name="text">
    <string>About RX12G Configure</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
